src/components/layout/Layout.spec.jsx

```diff
@@ -130,3 +141,15 @@ describe('LayoutBox', () => {
     expect(layoutBox.hasClass('sg-layout__box')).toEqual(true);
   });
 });
+
+describe('LayoutLeftContent', () => {
```

Comment:
Do we always render the contents of the left-most column? What happens on small screens?
```
L C R
```
v
```
L
C
R
```
?

Reply:
@tomasz-sodzawiczny we should display:

```
L
C
R
```